src/cr/cube/stripe/assembler.py

```diff
@@ -134,6 +134,14 @@ def scale_stderr(self):
         """
         return self._measures.scaled_counts.scale_stderr
 
+    @lazyproperty
+    def sum(self):
+        """1D np.float64 ndarray of sum for each row.
+        The array is filled with `np.nan` when the cube-result does not include a sum
+        cube-measure.
+        """
```

Comment:
The array is not filled with `np.nan` when there's no cube-sum measure, is it? It raises `ValueError` further down in that case. If docstring goes past the summary line it needs a blank line between.